plugins/maximize/plugin.js

```diff
@@ -113,24 +120,62 @@
 			var outerScroll;
 
 			// Saved resize handler function.
-			function resizeHandler() {
-				var viewPaneSize = mainWindow.getViewPaneSize();
-				editor.resize( viewPaneSize.width, viewPaneSize.height, null, true );
+			function resizeHandler( height ) {
+				var viewPaneSize = mainWindow.getViewPaneSize(),
+					panelHeight = checkIOS() && height ? height : viewPaneSize.height;
+
+				editor.resize( viewPaneSize.width, panelHeight, null, true );
+			}
+
+			function setContainerHeight( container ) {
+				// We must wait for end of autoZoom on iOS.
+				if ( checkIOS( container ) ) {
+					setTimeout( function() {
+						resizeHandler( window.innerHeight );
+
+						// After autoZoom we have to take the caret position and scroll the editable to the current position.
+						var html = editor.editable().getParent(),
+							scrollPos = mainWindow.getScrollPosition(),
+							toolbarHeight = html.getClientRect( true ).top;
+
+						if ( scrollPos.y !== 0 ) {
+							window.scrollTo( 0, 0 );
```

Comment:
Is this call really needed here? Since there is similar call few lines below - `window.scrollTo( scrollPos.x, 0 );`?